script.audio.profiles/default.py

```diff
@@ -6,187 +6,149 @@
 import xbmcvfs
 import json
 import os
+import sys
 
-__addon__ = xbmcaddon.Addon()
-__addon_id__ = __addon__.getAddonInfo('id')
-__addonpath__ = xbmc.translatePath(__addon__.getAddonInfo('path'))
-__datapath__ = xbmc.translatePath(os.path.join('special://profile/addon_data/', __addon_id__)).replace('\\', '/') + '/'
-__path_img__ = __addonpath__ + '/images/'
-__lang__ = __addon__.getLocalizedString
+__addon__               = xbmcaddon.Addon()
```

Comment:
Can you explain what exactly you mean?

2016-05-12 20:06 GMT+02:00 Philipp Temminghoff <notifications@github.com>:

> In script.audio.profiles/default.py
> <https://github.com/xbmc/repo-scripts/pull/153#discussion_r63068293>:
>
> >
> > -__addon__ = xbmcaddon.Addon()
> > -__addon_id__ = __addon__.getAddonInfo('id')
> > -__addonpath__ = xbmc.translatePath(__addon__.getAddonInfo('path'))
> > -__datapath__ = xbmc.translatePath(os.path.join('special://profile/addon_data/', __addon_id__)).replace('\\', '/') + '/'
> > -__path_img__ = __addonpath__ + '/images/'
> > -__lang__ = __addon__.getLocalizedString
> > +__addon__               = xbmcaddon.Addon()
>
> for future versions, please get rid of __varname__ variable naming.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/xbmc/repo-scripts/pull/153/files/3a6146d9a2c80fa46fbd70bc26334c2f5a51f378#r63068293>
>